identity/build.gradle

```diff
@@ -56,7 +57,7 @@ dependencies {
     // Needed for createComposeRule, but not createAndroidComposeRule:
     debugImplementation("androidx.compose.ui:ui-test-manifest:$androidxComposeVersion")
     // temporary fix for running compose test in RobolectricTestRunner, see https://github.com/robolectric/robolectric/issues/6593
-    testImplementation "androidx.test.espresso:espresso-core:3.5.0-alpha07"
+    testImplementation "androidx.test.espresso:espresso-core:3.5.0-beta01"
```

Comment:
Do you need something from this specific version? There's a `$espressoVersion` on the root build.gradle.

Reply:
Yes, as noted in the comment, we need a special version of espresso as all the compose tests for identity are written as Robolectric test as opposed to real AndroidTest, this version of expresso would make sure some of the mocked Android components from Robolectric is working correctly.